lib/ws-channel.js

```diff
@@ -289,11 +294,13 @@ WebsocketChannel.prototype._closeWebsocket = function(websocket) {
 
 
 WebsocketChannel.prototype.notify = function(message) {
-  this._debug('notify: %s', this._debug.json(message));
+  var requestId = this._requestId++;
+
+  this._debug('queue not %d: %s', requestId, this._debug.json(message));
```

Comment:
Its actually the packet type being queued: https://github.com/strongloop/strong-control-channel/blob/master/lib/ws-channel.js#L16, and used two lines below (`type: NOT`), but you are right that that is not clear, I uppercased it to make it more obvious that its a token, I hope.